### PR TITLE
Feature/solved routing and test errors

### PR DESCRIPTION
**Pull request by Tahmid11**
Routing:
- To make it compatible with GOV UK 5, changed the name to 'admin.users.show'

Permissions:
- Changed the permission function to return false (originally was 'manage'). This can be changed later 

User.test.php:
- Changed within testWithTitle(),  'Activity log of...' to 'Activities performed by...'

The Error of too few arguments:
- Called models and passed in a string representation of model to the actions function

### CODE REVIEW

**Comment by Tahmid11** [comment]
> Routing:
> 
>     * To make it compatible with GOV UK 5, changed the name to 'admin.users.show'
> 
> 
> Permissions:
> 
>     * Changed the permission function to return false (originally was 'manage'). This can be changed later
> 
> 
> User.test.php:
> 
>     * Changed within testWithTitle(),  'Activity log of...' to 'Activities performed by...'
> 
> 
> The Error of too few arguments:
> 
>     * Called models and passed in a string representation of model to the actions function

@SimRoberts9 Could I remove the line: 'config()->set('govuk-activity-log.user_model', $model);' since it's setting a key which is already set in the config itself?

**Comment by SimRoberts9** [comment]
@Tahmid11 I spoke with Ambreen about this after the call, you should be able to remove this.

There was a testing-trait-user-model in your TestCase file that it should pick up.

**Review by SimRoberts9 — APPROVED** [review]

File: src/ActivityLogServiceProvider.php [hunk 1]
```diff
@@ -51,11 +51,12 @@ protected function bootRoutes(): void
             Route::get('/{id}/activities', [
                 'as' => 'activities',
                 'class' => $model,
-                'uses' => ActivityController::class . '@activities',
+                'uses' => ActivityController::class . '@activity',
```
Comment (calum-gray) [inline src/ActivityLogServiceProvider.php:54]: why was this changed?

File: src/ActivityLogServiceProvider.php [hunk 2]
```diff
@@ -51,11 +51,12 @@ protected function bootRoutes(): void
...
             ]);
         });
     }
 
+
```
Comment (calum-gray) [inline src/ActivityLogServiceProvider.php:59]: blank lines

File: tests/Unit/ActivityController/ActionsTest.php [hunk 3]
```diff
@@ -25,8 +25,12 @@ protected function setUp(): void
             ->by($this->user)
             ->log('Toot');
 
+        $model = '\NetworkRailBusinessSystems\ActivityLog\Tests\Models\User';
+        config()->set('govuk-activity-log.user_model', $model);
+
```
Comment (calum-gray) [inline tests/Unit/ActivityController/ActionsTest.php:30]: blank lines

File: tests/Unit/User/UserTest.php [hunk 4]
```diff
@@ -48,7 +48,7 @@ public function testWithBack(): void
 
     public function testWithShowSubject(): void
     {
-        $this->assertFalse($this->response->getData()['showSubject']);
+        $this->assertTrue($this->response->getData()['showSubject']);
```
Comment (calum-gray) [inline tests/Unit/User/UserTest.php:51]: why have you changed this?

**Review by calum-gray — CHANGES_REQUESTED** [review]

File: src/ActivityLogServiceProvider.php (hunk 1 above)
Comment (SimRoberts9) [reply src/ActivityLogServiceProvider.php]: I don't recall when/why this was changed, let me and Ambreen know if you encounter any issues with this

File: tests/Unit/User/UserTest.php (hunk 4 above)
Comment (SimRoberts9) [reply tests/Unit/User/UserTest.php:51]: This was changed because the permission() function was altered to return false instead of the string 'manage'.

Regarding the code below, we are feeding in false and showing the subject.
```
        if ($permission !== false) {
            $this->authorize($permission, $subject);
        }
        return $subject;
```
@AmbreenJamal apologies I can't think of the wording you used to explain this on friday, would you mind adding in a comment, in relation to why we check for for false within the library?
